### DISCUSSION TITLE
More biome-unique land features / Option to de-select undesired biomes on New Game start

### DISCUSSION THREAD

**Discussion by RazzleDazzleDaeris** [voxeltycoon/issues#374] [General]
Hi team, I love the game!

Early on I find myself restarting every game until I get the Temperate biome, mainly because I love the snow-capped mountains, the big lakes, the flat valleys, and the overall uniqueness and challenge that the Temperate biome presents. I think the other biomes are great in theory, but don't seem to have the same uniqueness that the mountains and valleys have with the Temperate biome.

I suggest adding in some unique land features only seen in certain biomes, such as:

Deserts - blunt canyons and plateaus, sandstone arches (think like Utah and Zion National Park in the US), and rock spires.

Arctic - Perhaps frozen lakes? Maybe some snow cave-like features, some mountains that open up a bit into "caves"

Savannah - Perhaps some Oasis-like features. Some small water-holes surrounded by a lot of trees and shrubs.

Deciduous forest - rivers that cut through the region

Sub-Arctic - unfortunately I have no ideas here.

Also, think the option to de-select certain biomes would be appealing to many people. If I had the option right now, I think I'd start a world where every tile was using Temperate.

**Comment by MajorFordson** [comment]
Interestingly I think the limited types of trees we're seeing works okay. The addition of one unique bush/grass prop to each biome would really cement it in my opinion. (the savannah acacia style trees also look a lot like australian gum trees which is nice!) Interesting how a stylised game like VT can "get away" with having few prop types yet still look cohesive and good.

**Comment by RazzleDazzleDaeris** [comment]
I think the unique trees are great, visually speaking. No complaints there! I just think the Temperate biome actually changes the way you approach the game in that biome, as it presents very unique challenges. I feel the other biomes don't present as much of a direct impact to your gameplay. With some new features like arches, canyons, and stone spires, the Desert biome could absolutely also change how you approach the gameplay. Plus, with it being voxels, I think most of us would be ok with them turning up some of the "extreme" terrain generations.

**Comment by victorleblais** [comment]
This is planned on the roadmap: https://voxeltycoon.xyz/roadmap
![image](https://user-images.githubusercontent.com/4365075/115534109-bb3c8880-a297-11eb-85e4-285246c5a105.png)

**Comment by RazzleDazzleDaeris** [comment]
Hi yes, I knew the landscape features were on the roadmap, but I was wondering/hoping if the other biomes would have unique features like Temperate does? And if we'd have the option to select exactly which biomes do and do not generate in the world.

Thanks so much for the game and all your work!
